### PR TITLE
[DNM-1177] Separate PHP and IDR 

### PR DESCRIPTION
**Pull request by hglohika**
Separate PHP and IDR. Also separated the disbursement channels.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +33,20 @@ This library is the abstraction of Xendit API for access from applications writt
...
-        -   [Get Disbursement by ID](#get-disbursement-by-id)
-        -   [Get Disbursement by External ID](#get-disbursement-by-external-id)
-        -   [Get Disbursement Available Banks](#get-disbursement-available-banks)
+    -   [IDR Disbursements](#idr-disbursements)
+        -   [Create an IDR Disbursement](#create-an-idr-disbursement)
+        -   [Create an IDR Batch Disbursement](#create-an-idr-batch-disbursement)
+        -   [Get an IDR Disbursement by ID](#get-an-idr-disbursement-by-id)
+        -   [Get an IDR Disbursement by External ID](#get-an-idr-disbursement-by-external-id)
+        -   [Get an IDR Disbursement Available Banks](#get-an-idr-disbursement-available-banks)
+    -   [PHP Disbursements](#php-disbursements)
```
Comment (xenditjoshjimenez) [inline README.md:42]: @hglohika can we add a bit of wording on these two:

`IDR Disbursements for Indonesia`
`PHP Disbursements for Philippines`

just so they don't confuse the PHP currency against PHP the language

Comment (xen-HendryZheng) [reply README.md:42]: Agree

Comment (hglohika) [reply README.md]: @xenditjoshjimenez fff28fd4c7f22fcf5f2a29a582db4086205449f2

**Review by xenditjoshjimenez — APPROVED** [review]

**Review by itzpranz — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hglohika) [reply README.md]: @xen-HendryZheng Help to review it again. Thank you.

**Review by xen-HendryZheng — APPROVED** [review]

**Pull request merged**